demos/common/cpp/models/include/models/detection_model_yolo.h

```diff
@@ -27,14 +27,14 @@ class ModelYolo : public DetectionModel {
     class Region {
     public:
         int num = 0;
-        int classes = 0;
+        size_t classes = 0;
```

Comment:
It is still constructed from `int clasess` in 
https://github.com/openvinotoolkit/open_model_zoo/blob/b418da60e0c6aeb749b7acc9a9078b9c886a844b/demos/common/cpp/models/src/detection_model_yolo.cpp#L184-L188

Reply:
Updated